celeri/solve_mcmc.py

```diff
@@ -156,7 +159,9 @@ def _elastic_component(
     )
     n_eigs = eigenvectors.shape[1]
 
-    raw = pm.Normal(f"elastic_eigen_raw_{mesh}_{kind}", shape=n_eigs)
+    # Use eigenvalue-based variance for proper Matérn GP prior
+    eigenvalue_std = np.sqrt(eigenvalues)
+    raw = pm.Normal(f"elastic_eigen_raw_{mesh}_{kind}", sigma=eigenvalue_std, shape=n_eigs)
```

Comment:
Please renormalize / set the sigma scale, so that the top eigenmode has unit std.